spec/models/review_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe Review, type: :model do
+  describe "#content" do
+    it 'validates presence' do
+      review = Review.new
+      review.validate
+      expect(review.errors[:content]).to include("a great book")
```

Comment:
Here the test will check the errors, so we're expecting an error message (the default one is "can't be blank") 
If we want to test for specific content I think we'd have to do something like :

`expect(review.content).to eq("a great book")`